modules/nested-tabs/widgets/nested-tabs.php

```diff
@@ -421,10 +421,13 @@ protected function register_controls() {
 				'name' => 'tabs_title_background_color',
 				'types' => [ 'classic', 'gradient' ],
 				'exclude' => [ 'image' ],
-				'selector' => "{$nested_tabs_heading_selector_class} .e-n-tab-title",
+				'selector' => "{$nested_tabs_heading_selector_class} > .e-n-tab-title",
```

Comment:
Possibly delete and restore to original version. See comment above.

Reply:
Please see comment above